pkg/operator/ceph/controller/spec.go

```diff
@@ -624,13 +622,18 @@ func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Contai
 		Name: logCollector,
 		Command: []string{
 			"/bin/bash",
-			"-c",
+			"-c", // Command to run
```

Comment:
Wait... shouldn't `-c` be the last arg?

Reply:
It's no problem since bash ibterprets the first non-option argument a command. However, moving this option to the last arg is better to avoid confusion.